migrations/versions/rev-2023-08-17-15:38:43-bce7acfe5a4f_.py

```diff
@@ -12,7 +12,7 @@
 
 # revision identifiers, used by Alembic.
 revision = 'bce7acfe5a4f'
-down_revision = '24271525a263'
+down_revision = '1af4cc4f7cd5'
```

Comment:
Why is a database migration necessary here? I don't see any changes to the database structure?

Reply:
Because we encountered the problem "Error: Multiple moving heads" when running "ci/circleci:test". So I checked and changed the version of this file

Reply:
Indeed, previous changes introduce a split tree, how did that happen?
Thanks for fixing it

Reply:
I saw this problem happen 6 months ago and don't know why this problem happened